python/doc/theory/meta_modeling/enumeration_strategy.rst

```diff
@@ -313,7 +332,7 @@ Anisotropic hyperbolic enumeration strategy
 | One might also consider enumeration strategies based on an
   *anisotropic* hyperbolic norm defined by:
 
-  .. math:: \|\idx\|_{\vect{w},q} \, \, = \, \, \left(\sum_{i=1}^{n_X} \; w_i\; \alpha^q \right)^{1/q}
+  .. math:: \|\idx\|_{\vect{w},q} \, \, = \, \, \left(\sum_{i=1}^{n_X} \; w_i\; \alpha_i^q \right)^{1/q}
```

Comment:
There is the same error in the definition of the hyperbolic enumeration function. 
This is at line 176 of the original file.